tests/test_tools.py

```diff
@@ -83,6 +83,8 @@ def test_create_tool(client: Union[LocalClient, RESTClient]):
 
     def print_tool(message: str):
         """
+        Example tool that prints a message
```

Comment:
@mattzh72 @sarahwooders FYI I think this was a bug in the test. If you looked through the test, `print_tool` was getting created with a null description, which I think we want to make impossible to happen because `description` is key to getting the LLM to understand the tool.

Reply:
I see - we had made description optional in the model before, so assumed it was okay to be null.